app/controllers/tutorials_controller.rb

```diff
@@ -68,6 +68,7 @@ def view
     if draft_check == true
       render "tutorial_draft"
     else
+      # TODO google rails determine if request is XHR
```

Comment:
Annotation keywords like `TODO` should be all upper case, followed by a colon, and a space, then a note describing the problem.